modules/postgresql/main.tf

```diff
@@ -35,7 +26,7 @@ resource "google_sql_database_instance" "default" {
     availability_type           = "${var.availability_type}"
     authorized_gae_applications = ["${var.authorized_gae_applications}"]
     backup_configuration        = ["${var.backup_configuration}"]
-    ip_configuration            = "${local.ip_configurations["${local.ip_configuration_enabled ? "enabled" : "disabled"}"]}"
+    ip_configuration            = ["${var.ip_configuration}"]
```

Comment:
Same as above.

Reply:
Restored the behavior, but I still had to change this line.

Without the '[...]', the example instantiation fails. This solution passes the integration tests.